src/installer/managed/Microsoft.NET.HostModel/Bundle/Bundler.cs

```diff
@@ -160,9 +160,12 @@ public string GenerateBundle(IReadOnlyList<FileSpec> fileSpecs)
                 throw new ArgumentException("Invalid input specification: Must specify the host binary");
             }
 
-            if (fileSpecs.GroupBy(file => file.BundleRelativePath).Where(g => g.Count() > 1).Any())
+            var duplicateGroups = fileSpecs.GroupBy(file => file.BundleRelativePath).Where(g => g.Count() > 1);
+
+            if (duplicateGroups.Any())
             {
-                throw new ArgumentException("Invalid input specification: Found multiple entries with the same BundleRelativePath");
+                throw new ArgumentException("Invalid input specification: Found multiple entries with the same BundleRelativePath:"
+                    + string.Join("\n", duplicateGroups.SelectMany(o => o).Select(o => "  " + o.BundleRelativePath)));
```

Comment:
If we print the duplicates here, I think we should print both the `SourcePath` and `BundleRelativePath` to identify the incorrect entry.

So, I suggest that 
* Add a `ToString()` function to [`FileSpec`](https://github.com/dotnet/runtime/blob/master/src/installer/managed/Microsoft.NET.HostModel/Bundle/FileSpec.cs) to print both paths. 
* Simplify the exception throw to something like

```
                throw new ArgumentException("Invalid input specification: Found multiple entries with the same BundleRelativePath:"
                        + string.Join(Environment.NewLine, duplicateGroups.SelectMany(o => o)));
```